torch/csrc/jit/frontend/builtin_functions.cpp

```diff
@@ -67,11 +67,65 @@ def list_with_default(out_size: List[int], defaults: List[int]):
 
 // Implementations of historic symbol behaviors are defined here
 // See note [Versioned Symbols]
+
+// This builtin is for testing
 auto _test_serialization_subcmul = R"SCRIPT(
 def _test_serialization_subcmul_0_2(self: Tensor, other:Tensor, alpha: number=2) -> Tensor:
   return other - (self * alpha)
 )SCRIPT";
 
+// Division versioned symbols, for Torchscript programs serialized when
+// division on integer tensors was floor division, not true division.
+// NOTE: updated division is not yet implemented. These versioned
+// symbols are here preemptively.
+
+// Tensor x Tensor
+auto div_tensor = R"SCRIPT(
+def div_0_3(self: Tensor, other: Tensor) -> Tensor:
+  if (self.is_floating_point() or other.is_floating_point()):
```

Comment:
curious why this doesn't match the implementation (i.e. use integral instead of floating point for the check)?  I don't remember how bool/complex relates to those in torchscript though.

Reply:
Torchscript doesn't support complex numbers, and since this is historic the programs it applies to won't ever deal with complex, so checking for float is OK and we don't have a similar function for integers + bools available.

Reply:
sounds good, let's add a comment to that effect.